.github/workflows/archive-top-measurements.yml

```
@@ -0,0 +1,40 @@
+on:
+  schedule:
+    - cron: "0 12 * * *" # Once per day
```

```suggestion
    - cron: "0 12,13 * * *"
```

We could make this round multiple times a day, so that when it fails it will be retried later

---

Yes, please!

---

We are refreshing the materialised view every 12 hours. Let's run this action at 1pm and 2pm to make sure the update has been executed and the execution finished.

```suggestion
    - cron: "0 13,14 * * *" # Once per day + one retry
```